src/elements/content-sidebar/SidebarNavButton.js

```diff
@@ -29,6 +30,7 @@ const SidebarNavButton = React.forwardRef<Props, React.Ref<any>>((props: Props,
         isOpen,
         sidebarView,
         tooltip,
+        'aria-label': dataAriaLabel,
```

Comment:
nit: variable name doesn't need `data` at the beginning, that's for the data-* attributes
Can we add a test or update the snapshots for this file showing that the DOM node receives the label?

Reply:
@alexkrolick I just changed the aria-label variable name and I also added an unit test in `SidebarNavButton.test.js `